src/nwb_conversion_tools/tools/roiextractors/roiextractors.py

```diff
@@ -205,6 +208,13 @@ def get_nwb_imaging_metadata(imgextractor: ImagingExtractor):
     metadata["Ophys"]["ImagingPlane"][0].update(imaging_rate=rate)
     # TwoPhotonSeries update:
     metadata["Ophys"]["TwoPhotonSeries"][0].update(dimension=imgextractor.get_image_size(), rate=rate)
+
+    device_name = metadata["Ophys"]["Device"][0]["name"]
+    metadata["Ophys"]["ImagingPlane"][0]["device"] = device_name
+
+    plane_name = metadata["Ophys"]["ImagingPlane"][0]["name"]
+    metadata["Ophys"]["TwoPhotonSeries"][0]["imaging_plane"] = plane_name
+
```

Comment:
Why not have this incorporated into `default_ophys_metadata()`? https://github.com/catalystneuro/nwb-conversion-tools/blob/dacddd4a485bce35cd68c97ef5355aaa84c1ff2c/src/nwb_conversion_tools/tools/roiextractors/roiextractors.py#L39

Reply:
I tried this. This function is also used in the segmentation pipeline and it causes problems over there. @CodyCBakerPhD mentioned that there is an incoming refactor of `write_segmentation`, I think we will be able to do it then.

Reply:
ah okay, sorry then, I haven't checked write_segmentation thoroughly yet.

Reply:
Is a good idea! let's revisit it when we are doing the refactor of `write_segmentation`.